app/mm-redux/actions/integrations.ts

```diff
@@ -13,7 +13,7 @@ import {analytics} from '@init/analytics.ts';
 
 import {batchActions, DispatchFunc, GetStateFunc, ActionFunc} from '@mm-redux/types/actions';
 
-import {Command, DialogSubmission, IncomingWebhook, OAuthApp, OutgoingWebhook} from '@mm-redux/types/integrations';
+import {Command, DialogSubmission, IncomingWebhook, OAuthApp, OutgoingWebhook, CommandArgs} from '@mm-redux/types/integrations';
```

Comment:
nit: keep import order alphabetical 
```suggestion
import {Command, CommandArgs, DialogSubmission, IncomingWebhook, OAuthApp, OutgoingWebhook} from '@mm-redux/types/integrations';
```